service/controller/clusterapi/v21/controllercontext/client.go

```diff
@@ -2,7 +2,6 @@ package controllercontext
 
 import (
 	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
-	"github.com/giantswarm/helmclient"
```

Comment:
Is there even more to drop now? What is with the vendor deps?

Reply:
Not till we can delete old version bundles or flatten the operator. The tiller resource is still present in v19 or older.

https://github.com/giantswarm/cluster-operator/tree/master/pkg/v19/resource/tiller